embed/config.go

```diff
@@ -611,7 +612,7 @@ func (cfg *Config) PeerURLsMapAndToken(which string) (urlsmap types.URLsMap, tok
 	case cfg.DNSCluster != "":
 		clusterStrs, cerr := cfg.GetDNSClusterNames()
 		lg := cfg.logger
-		if cerr != nil {
+		if len(clusterStrs) == 0 && cerr != nil {
 			lg.Warn("failed to resolve during SRV discovery", zap.Error(cerr))
```

Comment:
Can we log the warning always when cerr != nil, 
but return from the method only when clusterStrs is empty.

Reply:
Done